composer.json

```diff
@@ -23,6 +23,7 @@
         "container-interop/container-interop": "^1.1",
         "vlucas/phpdotenv": "^2.2",
         "joomla/registry": "^1.5",
+        "aura/session": "^2.0",
```

Comment:
Joomla framework is using the same namespace and is marked as deprecated. Symfony has session management which is part of the foundation library, means extra bloat is loaded.
But as you recon, it doesn't matter which lib is used as it is behind an interface anyway, to be able to change it later to the dgt41 session library :see_no_evil: